### ISSUE TITLE
module timing

### ISSUE THREAD

**Issue by renikaul** [anabento/R_Bootcamp#2]
I think the afternoon of day 2 is a bit too ambitious. What about moving the M7 to before lunch and M6 to before the break?

Are breaks in the morning and afternoon ok?

**Comment by anabento** [comment]
Yes breaks should be there, otherwise they might explode. Yes day 2 is ambitious, we can swap it at least for now and check again the order once they are written
